From 23b2b4e28940218bfa3832c4edc6f535f5f67aa0 Mon Sep 17 00:00:00 2001
From: Giovanni Campagna <gcampagna@src.gnome.org>
Date: Thu, 16 Aug 2012 14:08:19 +0000
Subject: power: fix DBus path of gnome-screensaver

The right path for both gnome-screensaver and gnome-shell is
/org/gnome/ScreenSaver. It was never noticed because the proxy
is created without properties and there is no callback in the
method calls.

https://bugzilla.gnome.org/show_bug.cgi?id=682007
---
diff --git a/plugins/power/gsd-power-manager.c b/plugins/power/gsd-power-manager.c
index 76e1a6e..31a20a9 100644
--- a/plugins/power/gsd-power-manager.c
+++ b/plugins/power/gsd-power-manager.c
@@ -73,7 +73,7 @@
 #define GSD_POWER_DBUS_INTERFACE_KEYBOARD       "org.gnome.SettingsDaemon.Power.Keyboard"
 
 #define GS_DBUS_NAME                            "org.gnome.ScreenSaver"
-#define GS_DBUS_PATH                            "/"
+#define GS_DBUS_PATH                            "/org/gnome/ScreenSaver"
 #define GS_DBUS_INTERFACE                       "org.gnome.ScreenSaver"
 
 #define GSD_POWER_MANAGER_NOTIFY_TIMEOUT_NEVER          0 /* ms */

